### PR TITLE
Fix birthday passed age displayed bug.

### LINKED ISSUE #196: Birthday age not calculated properly

### PR DESCRIPTION
**Pull request by Justinhoejj**
Fixes #196. Increment age displayed when birthday has passed.

### CODE REVIEW

**Review by yucheng11122017 — APPROVED** [review]
LGTM!

**Review by Jiwei99 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/ui/BirthdayReminderCard.java [hunk 1]
```diff
@@ -82,7 +82,12 @@ private String generateBirthdayReminderMessage(Birthday birthday) {
             cardPane.setStyle(STYLE_CARD_BIRTHDAY_THIS_WEEK);
             return String.format(BIRTHDAY_MESSAGE_THIS_WEEK, age, daysToBirthday, birthMonthDayAsString);
         }
-        return String.format(BIRTHDAY_MESSAGE_DEFAULT, age, birthMonthDayAsString);
+        if (daysToBirthday < 0) {
+            // birthday has passed increment age for displaying next year's age
+            return String.format(BIRTHDAY_MESSAGE_DEFAULT, age + 1, birthMonthDayAsString);
+        } else {
+            return String.format(BIRTHDAY_MESSAGE_DEFAULT, age, birthMonthDayAsString);
```
Comment (cookiedan42) [inline src/main/java/seedu/address/ui/BirthdayReminderCard.java:89]: consider putting the 4 items in a if else if stack so its obvious that they are exclusive paths?

**Review by cookiedan42 — APPROVED** [review]
Code style, but logic is sound

**Pull request merged**